installation/on_mac_osx_using_homebrew.md

```diff
@@ -21,3 +21,27 @@ you need to reinstall the command line tools and then select the default active
 $ xcode-select --install
 $ xcode-select --switch /Library/Developer/CommandLineTools
 ```
+
+## Troubleshooting on macOS MacOS 10.14.2 (Mojave)
+
+If you get an error like:
+
+```
+ld: library not found for -lssl (this usually means you need to install the development package for libssl)
+```
+
+you may need to install OpenSSL and link pkg-config to OpenSSL
+
+```
+brew install openssl
+```
+
+```
+export PKG_CONFIG_PATH=$PKG_CONFIG_PATH:/usr/local/opt/openssl/lib/pkgconfig
+```
+
+```
+As with other keg-only formulas there are some caveats shown in brew info <formula> that advertise how to link pkg-config with this library.
```

Comment:
```suggestion
As with other keg-only formulas there are some caveats shown in `brew info <formula>` that shows how to link pkg-config with this library.
```

Reply:
@dustinlahr was the commit batch feature not available? It allows you to squash all suggestions into one commit.

Reply:
I did not see it, so I just did it individually.

Reply:
No worries!